src/components/UserDisplay.js

```diff
@@ -0,0 +1,16 @@
+import UserInfo from "./searchResults/UserInfo";
+import UserAccount from "./searchResults/UserAccount";
+import UserContact from "./searchResults/UserContact";
+import "./UserDisplay.css";
+
+const UserDisplay = (props) => {
+  return (
+    <div className="user-display">
+      <UserInfo userInfo={props.userData.profile} />
+      <UserAccount accountInfo={props.userData.account} />
+      <UserContact icons={props.icons} contactInfo={props.userData.contact} />
+    </div>
+  );
+};
+
```

Comment:
```suggestion
const UserDisplay = ({userData, icons}) => {
const { profile, account, contact } = userData;
  return (
    <div className="user-display">
      <UserInfo userInfo={profile} />
      <UserAccount accountInfo={account} />
      <UserContact icons={icons} contactInfo={contact} />
    </div>
  );
};
```


First, how you have it written is correct - and it works ✨. 

We can however improve it by destructuring. This improves the code readability and also you do not have to repeat writing `this.props` often. If this seems complex at first sight - that's alright. You can read more about [props destructuring in React.](https://www.geeksforgeeks.org/destructuring-of-props-in-reactjs/)

Destructuring is a Javascript feature that helps to unpack the content in an object or array. You can read[ more here](https://javascript.info/destructuring-assignment)

Reply:
Yes, I am good with destructuring and it all makes sense.  Effecting that

Reply:
Changed. I one-upped you and did a nested destructuring instead 😝